src/main/java/com/gruzini/tennistico/events/ChangeMatchStatusEvent.java

```diff
@@ -10,12 +11,17 @@
     private final LocalDateTime expirationDateTime;
     private final MatchStatus currentMatchStatus;
     private final MatchStatus desiredMatchStatus;
+    private final DateTimeSelector dateTimeSelector;
 
-    public ChangeMatchStatusEvent(Object source, LocalDateTime expirationDateTime, MatchStatus currentMatchStatus, MatchStatus desiredMatchStatus) {
+    public ChangeMatchStatusEvent(Object source, LocalDateTime expirationDateTime,
+                                  MatchStatus currentMatchStatus,
+                                  MatchStatus desiredMatchStatus,
+                                  DateTimeSelector dateTimeSelector) {
```

Comment:
Słabo, że aż 4 argumenty. Wymyślmy refactor jakiś na to.

Reply:
tego nawet nie zmieniałem, rozbiłem na wiele linii po prostu argumenty : )

Reply:
ja bym optował żeby użyć jednej i tej samej metody, imo nie ma znaczenia czy bedziemy liczyli od startu czy od konca. liczmy od startu to się od razu to uprości